src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -267,6 +285,10 @@ UniversalNavbarExpanded.propTypes = {
     href: PropTypes.string,
     title: PropTypes.string,
   }),
+  /** Add color to nav bar*/
+  navbarColor: PropTypes.string,
```

Comment:
I think if we could go with just a transparent background to start, when `animateNavbar` is set to `true`, then we could do away with this `navbarColor` prop. 

It would make it easier to work with on the CMS, since we wouldn't have to update EDS anytime we want to use a different background color.